src/zeit/solr/tests/test_converter.py

```diff
@@ -142,3 +142,42 @@ def test_print_ressort_is_indexed(self):
         self.assertEqual(
             'Feuilleton',
             xml.xpath('//field[@name="ns-print-ressort"]')[0].text)
+
+    def test_volume_is_indexed(self):
+        import datetime
+        import pytz
+        import zeit.content.volume.volume
+        volume = zeit.content.volume.volume.Volume()
+        volume.uniqueId = 'http://xml.zeit.de/volume'
+        volume.year = 2015
+        volume.volume = 1
+        volume.product = zeit.cms.content.sources.Product(u'ZEI')
+        volume.date_digital_published = datetime.datetime(
+            2015, 1, 1, 0, 0, tzinfo=pytz.UTC)
+        xml = self.convert(volume)
+        self.assertEqual('volume', xml.xpath('//field[@name="type"]')[0].text)
+        self.assertEqual(
+            volume.uniqueId, xml.xpath('//field[@name="uniqueId"]')[0].text)
```

Comment:
In assertions generell lieber duplizieren, also explizit `'http://xml.zeit.de/volume'` hinschreiben.